### ISSUE TITLE
zenpy 2.0.27 is not installs dependencies of require_installs

### ISSUE THREAD

**Issue by hassaku63** [facetoe/zenpy#554]
Even though I installed zenpy 2.0.27, it seems that its required dependencies weren't installed (such as cachetools).

It seems that the required dependencies are installed in version 2.0.16.

## Reproduce

Runtime and pip version is here.

OS: MacOS 13.4.1

```bash
$ python -V                                            
Python 3.9.13

$ pip --version
pip 23.2.1 from <venv-project-path>(python 3.9)
```

```
$ pip freeze 

$ pip install zenpy==2.0.25
...

$ pip freeze                                           
cachetools==5.3.1
certifi==2023.7.22
charset-normalizer==3.2.0
idna==3.4
python-dateutil==2.8.2
pytz==2023.3
requests==2.31.0
six==1.16.0
urllib3==2.0.4
zenpy==2.0.25

$ pip uninstall -y $(pip freeze | awk -F= '{print $1}')
...

$ pip install zenpy==2.0.27
...

$ pip freeze                                         
zenpy==2.0.27
```

## workaround

I will avoid use this version.

example: 

```
# requirements.txt
zenpy~=2.0.0, !=2.0.27
```

**Comment by modprobe** [comment]
We ran into this issue as well and had to pin to `2.0.25` for now. I can't spot anything obvious in the diff between those two tags, but I'm also not an expert in python packaging

**Comment by Ptr314** [comment]
Thank you for the reporting! I think we've fixed it (mostly @cryptomail). Could you please try 2.0.30?

**Comment by cryptomail** [comment]
Issue should be fixed with 2.0.30.  Sorry for any duress we have caused you.  :(

**Comment by estahn** [comment]
@Ptr314 Would you mind sharing the cause? I can't see any changes in the files https://github.com/facetoe/zenpy/compare/2.0.27...2.0.30 ... is that something that is generated during the CI process?
